### PR TITLE
HPCC-15873 Add FileUploadAccess to default permissions

### PR DESCRIPTION
**Pull request by wangkx**
When ESP SMC service is started, the service reads feature
permission settings from esp.xml and add the permission to
LDAP server if the permission is not in the LDAP server.
In this fix, FileUploadAccess is added to environment
configuration files which creates the FileUploadAccess
permission setting in esp.xml.

Signed-off-by: wangkx kevin.wang@lexisnexis.com

### CODE REVIEW

**Comment by wangkx** [comment]
@RussWhitehead please review.

File: initfiles/componentfiles/configxml/buildsetCC.xml.in [hunk 1]
```diff
@@ -162,6 +162,10 @@
                           path="FileDkcAccess"
                           resource="FileDkcAccess"
                           service="ws_fs"/>
+     <AuthenticateFeature description="Access to upload files to LZ"
```
Comment (RussWhitehead) [inline initfiles/componentfiles/configxml/buildsetCC.xml.in]: Should we spell out "Landing Zone"  ( see the feature right below this, it reads "Access to files in dropzone")

**Comment by RussWhitehead** [comment]
@wangkx Review complete, looks good, just one minor usability question

File: initfiles/componentfiles/configxml/buildsetCC.xml.in (hunk 1 above)
Comment (wangkx) [inline initfiles/componentfiles/configxml/buildsetCC.xml.in]: I guess I should change it to 'dropzone'.

**Comment by wangkx** [comment]
@richardkchapman this PR is ready to be merged to 6.0.2.

**Comment by richardkchapman** [comment]
Too late for 6.0.2

Is this really a point release change, or should it wait for 6.2 ?

**Comment by wangkx** [comment]
How about 6.0.4? I think it should be a point release change.

**Comment by richardkchapman** [comment]
@wangkx  6.0.4 then

**Comment by wangkx** [comment]
I am closing this PR. A new PR (https://github.com/hpcc-systems/HPCC-Platform/pull/8880) has been opened for 6.0.4.

**Pull request closed without merging**